### PR TITLE
[ENH] New WarpPoints interface in algorithms.mesh

### PR DESCRIPTION
**Pull request by oesteban**
An interface to Warp vtk meshes from a displacement field. Uses scipy.map_coordinates interpolation.

EDIT: this interface is much faster than fsl.WarpPoints in the sense that the coefficients (for bspline interpolation) are computed only once. Conversely to fsl.WarpPoints, this interface only accepts points in physical space coordinates

- [x] WarpPoints interface
- [x] Properly describe where the displacements fields come from (i.e. which other interfaces are compatible).

### CODE REVIEW

**Comment by oesteban** [comment]
I have added a more detailed docstring for the interface. I hope this fulfills the need for a more detailed description of compatible warping fields.

File: nipype/algorithms/mesh.py [hunk 1]
```diff
@@ -25,11 +25,143 @@
...
+        return op.abspath('%s_%s.%s' % (fname, suffix, ext))
+
+    def _run_interface(self, runtime):
+        vtk_major = 6
+        try:
+            import vtk
+            vtk_major = vtk.VTK_MAJOR_VERSION
+        except ImportError:
+            iflogger.warn(('python-vtk could not be imported'))
+            pass
```
Comment (chrisfilo) [inline nipype/algorithms/mesh.py]: Codacy Issue found: [Unnecessary pass statement](https://www.codacy.com/app/krzysztof-gorgolewski/nipy_nipype/file/825127397/issues/source?bid=2767207&fileBranchId=2767503#l89)

File: nipype/algorithms/mesh.py [hunk 2]
```diff
@@ -25,11 +25,143 @@
...
+            from tvtk.api import tvtk
+        except ImportError:
+            raise ImportError('Interface requires tvtk')
+
+        try:
+            from enthought.etsconfig.api import ETSConfig
+            ETSConfig.toolkit = 'null'
+        except ImportError:
+            iflogger.warn(('ETS toolkit could not be imported'))
+            pass
```
Comment (chrisfilo) [inline nipype/algorithms/mesh.py]: Codacy Issue found: [Unnecessary pass statement](https://www.codacy.com/app/krzysztof-gorgolewski/nipy_nipype/file/825127397/issues/source?bid=2767207&fileBranchId=2767503#l101)

File: nipype/algorithms/mesh.py [hunk 3]
```diff
@@ -25,11 +25,143 @@
...
+        except ValueError:
+            iflogger.warn(('ETS toolkit could not be set to null'))
+            pass
```
Comment (chrisfilo) [inline nipype/algorithms/mesh.py]: Codacy Issue found: [Unnecessary pass statement](https://www.codacy.com/app/krzysztof-gorgolewski/nipy_nipype/file/825127397/issues/source?bid=2767207&fileBranchId=2767503#l104)

**Pull request merged**